runironic-api.sh

```diff
@@ -2,4 +2,13 @@
 
 . /bin/configure-ironic.sh
 
+# Wait for ironic to load all expected drivers
+# the DB query returns a string like ["fake-hardware", "idrac", "ipmi"]
+DB_DRIVERS=$(mysql --user=ironic --password=${MARIADB_PASSWORD} --protocol=tcp -r -s -e "use ironic; select drivers from conductors;" | jq '.|length')
+CONF_DRIVERS=$(crudini --get ironic.conf DEFAULT enabled_hardware_types tr ',' '\n' | wc -l)
```

Comment:
we're need to cd to the /etc/ironic directory here or specificy the full path to ironic.conf

also missing a "|" before the tr

Reply:
Thanks, done